core/pkg/artifacts/linker.go

```diff
@@ -72,3 +81,50 @@ func (al *ArtifactLinker) Link() error {
 
 	return err
 }
+
+// resolveOrgEntityName take an Entity and an potentially empty Organization(display org name or org entity name)
+// and using those inputs we fetch the org entity and:
+// 1. validate the user inputted the correct org name or org entity name for the link path
+// 2. return the org entity name in place of if no org was inputted or the display org name
+// so we can use the correct entity name to link the artifact.
+func (al *ArtifactLinker) resolveOrgEntityName(portfolioEntity string, organization string) (string, error) {
+	orgFieldNames, err := GetGraphQLFields(al.Ctx, al.GraphqlClient, "Organization")
+	if err != nil {
+		return "", err
+	}
+	canFetchOrgEntity := slices.Contains(orgFieldNames, "orgEntity")
+	if organization == "" && !canFetchOrgEntity {
+		return "", fmt.Errorf("Fetching Registry artifacts without inputting an organization is " +
+			"unavailable for your server version. Please upgrade your server to 0.50.0 or later.")
```

Comment:
This error isn't surfaced to the user. The conventional format is lowercase with no punctuation at the end, so

```suggestion
		// Support is added in version 0.50.0 of the wandb server.
		return "", fmt.Errorf("fetching Registry artifacts unsupported and no organization given")
```